packages/teams-js/src/internal/communication.ts

```diff
@@ -23,6 +30,7 @@ export class Communication {
   public static parentWindow: Window | any;
   public static childWindow: Window;
   public static childOrigin: string;
+  public static alreadyVerifiedDomains: Map<string, boolean> = new Map();
```

Comment:
The name of this threw me off a little: in my head "Verified" meant "domains that we have already checked and found valid", but what it actually means is "domains that we have already checked and what the result of that check was". I might be overthinking naming (shocking, I know), but maybe something like `alreadyCheckedDomains` instead?

Reply:
I was thinking something similar, since validated and verified imply that these are the domains that passed, when the reason this exists is to not bog down calls to our service endpoint with a list of known results. i.e `example.com: false`, but `example.com` keeps sending messages to the app, we shouldn't keep having to verify it if we know that it is incorrect.